src/Sylius/Behat/Context/Setup/CustomerContext.php

```diff
@@ -125,18 +145,42 @@ public function theStoreHasCustomerAccountWithEmailAndPassword($name, $email, $p
     }
 
     /**
+     * @Given /^(he) has shipping (address as "(?:[^"]+)", "(?:[^"]+)", "(?:[^"]+)", "(?:[^"]+)" for "(?:[^"]+)")$/
+     */
+    public function heHasShippingAddress(CustomerInterface $customer, AddressInterface $address)
+    {
+        $customer->setShippingAddress($address);
+
+        $this->customerManager->flush($customer);
```

Comment:
The `ObjectManager` interface has only `flush()` method that doesn't accept any argument. `flush($customer)` should be changed to `flush()`